source/documentation/internal/guest-agent/architecture.html.md

```diff
@@ -14,16 +14,15 @@ wiki_last_updated: 2013-05-20
 
 ## The communication between oVirt guest agent and the oVirt engine
 
-![](Guest_Agent_Achitecture.png "Guest_Agent_Achitecture.png")
+![](/images/wiki/Guest_Agent_Achitecture.png)
 
 The engine directs requests for the guest agent to VDSM.
 
 VDSM communicates with the guest using a VirtIO channel using plain text commands in JSON format. Each command is terminated by a new line character.
 
 ## VDSM/Guest Agent message structure
 
-Each message sent by VDSM has a mandatory field '__name__' which contains the command name.
+Each message sent by VDSM has a mandatory field `__name__` which contains the command name.
```

Comment:
Backticks are used in Markdown for inline code/keywords, etc. Comapre: 'this' and `this`